### PR TITLE
Updates rspec syntax.

### LINKED ISSUE #24: Update specs to play nice with rspec 3 and latest equivalent-xml
**Done when**
  1. All `\.should`  syntax replaced with `expect().to`
  1. No errors due to missing rspec matchers

### PR DESCRIPTION
**Pull request by afred**
Adds additional requirement by equivalent-xml gem. Closes #24.

### CODE REVIEW

**Comment by afred** [comment]
Travis failing b/c https://github.com/projecthydra/active_fedora/pull/766. Re-run the build after the PR is merged.

**Comment by jeremyf** [comment]
@afred I relaunched the Travis build.

**Comment by jeremyf** [comment]
Actually, I launched a completely unrelated build. This repository has some odd entanglements on Travis. Its still pointing to @awead's build environment.

**Comment by afred** [comment]
Travis is confused because we moved the repo on him. Not yet sure how to correct this.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # :nodoc
 require "hydra-pbcore"
+require 'rspec/matchers'
 require "equivalent-xml"
-require "pry"
+# require "pry"
```
Comment (awead) [inline spec/spec_helper.rb]: change this to
```
require 'pry' unless ENV['TRAVIS']
```

**Comment by afred** [comment]
Hmm... tried force pushing to get Travis to run build from the `projecthydra-labs/hydra-pbcore` repo instead of `awead/hydra-pbcore`... but it doesn't appear to have triggered. Going to close and resubmit the PR, see if that works.

**Comment by afred** [comment]
Hm.. that didn't work... see #31.

**Pull request closed without merging**